src/components/PlayerDash.jsx

```diff
@@ -62,6 +73,41 @@ export const PlayerDash = React.createClass({
       });
 
     });
-  }
 
+    this.firebase.getEloDateForCurrentMonth(this.props.params.playerId, 'value',(rawItems) => {
+      var items = [];
+
+      rawItems.forEach( (rawItem) => {
+        var item = rawItem.val();
+        item.id = rawItem.key();
+        items.push(item);
+      });
+
+      var getDataPoint = function getDataPoint(id, item){
```

Comment:
Cool man. I just read about let again. Makes sense. I think I was just used to C# from work.